panther_lights/test/test_animation.cpp

```diff
@@ -74,7 +74,7 @@ TEST_F(TestAnimation, Initialize)
   animation_description["duration"] = "-1.0";
   EXPECT_THROW(animation_->Initialize(animation_description, 10, 10.0), std::out_of_range);
   animation_description["duration"] = "word";
-  EXPECT_THROW(animation_->Initialize(animation_description, 10, 10.0), YAML::BadConversion);
+  EXPECT_THROW(animation_->Initialize(animation_description, 10, 10.0), std::runtime_error);
 
   // invalid animation length
```

Comment:
If this test means that the controller period is longer than the animation length, this description seems to make more sense to me. In this case I would also set the smallest possible range that should cause an error.
```suggestion
  // invalid controller frequency
```

Reply:
Not exactly, animation length is an unsigned value that represents number of frames that animation will have. It depends on both controller frequency and animation duration. If length is `< 1`, then it should thrown an exception as there are no frames to be displayed

Reply:
If I understand correctly, the animation must absolutely last at least 1 second? If so, I am missing this information in the readme